U/utf8proc/build_tarballs.jl

```diff
@@ -18,7 +18,7 @@ if [[ "${target}" == *-mingw* ]]; then
     mkdir -p ${libdir} ${prefix}/lib ${includedir}
     cp utf8proc.h ${includedir}
     ar x libutf8proc.a
-    cc -shared -o "${libdir}/libutf8proc.${dlext}" *.o
+    $CC -shared -o "${libdir}/libutf8proc.${dlext}" *.o
```

Comment:
`$CC` is `cc`, so I'm very confused by this change.

Reply:
So `cc` will point to the correct compiler even when cross-compiling?

Reply:
`cc` _is_ the cross-compiler